src/api/.rubocop_todo.yml

```diff
@@ -815,6 +815,28 @@ Rails/AssertNot:
     - 'test/unit/role_test.rb'
     - 'test/unit/user_test.rb'
 
+# Offense count: 24
+# Configuration parameters: Database, Include.
+# SupportedDatabases: mysql, postgresql
+# Include: db/migrate/*.rb
+Rails/BulkChangeTable:
+  Exclude:
```

Comment:
I wonder if we want to modify old migrations to fix those offenses, if not, instead of regenerating the rodo file we should exclude this files in the `.rubocop.yml` file. I would personally modify old migrations, but it doesn't make such a different and I think @ChrisBr was again doing this.... :thinking:

Reply:
@Ana06, I prefer to exclude them.